docs/cli.md

```diff
@@ -523,10 +523,33 @@ gcli role list
 
 ```bash
 gcli role create --role admin
- ```
+```
 
 #### Delete a role
 
 ```bash
 gcli role delete --role admin
- ```
+```
+
+#### Add a role to a user
+
+```bash
+gcli user update --metalake metalake_demo --user new_user --role admin
+```
+
+#### Remove a role from a user
+
+```bash
+gcli user delete --metalake metalake_demo --user new_user --role admin
```

Comment:
Btw, please remove the "--metalake" from the samples.

Reply:
Done